tests/apps/courses/test_cms_plugins_category.py

```diff
@@ -22,6 +22,33 @@ class CategoryPluginTestCase(CMSTestCase):
     Test that CategoryPlugin correctly displays a Category's page placeholders content
     """
 
+    @staticmethod
+    def test_cms_plugins_category_unique():
```

Comment:
I think it would be clearer to remove this test and modify the above test. You just need to replace:
```python
self.assertIn(leaf_category.extended_object.get_title(), rendered_form)
```
by 
```python
self.assertEqual(rendered_form.count(leaf_category.extended_object.get_title()), 1)
```

and update the docstring...

Reply:
done